src/FsUnit.MbUnit/FsUnit.fs

```diff
@@ -27,7 +30,8 @@ let equal expected = CustomMatchers.equal expected
 
 let equalWithin (tolerance:obj) (expected:obj) = CustomMatchers.equalWithin tolerance expected
 
-let not' (expected:obj) = CustomMatchers.not' expected
+let not' (expected:obj) =
+    if box expected = null then CustomMatchers.not' (IsNull()) else (CustomMatchers.not' expected)
```

Comment:
I think that we could move implementation of `not`` and fix to `equal` to `CustomMatchers.fs` file that should be linked from all projects and do not copy parse implementation.